apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.html.ts

```diff
@@ -271,50 +251,41 @@ export function InstructorAssessmentStatistics({
 
           <div class="card mb-4">
             <div class="card-header bg-primary text-white">
-              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Score statistics by
-              date
+              ${resLocals.assessment_set.name} ${assessment.number}: Score statistics by date
             </div>
 
-            ${resLocals.assessment.score_stat_number > 0
+            ${assessment.score_stat_number > 0
               ? html`
                   <div class="card-body">
                     <div
                       id="scoreHistsByDateDiv"
                       style="overflow-x: scroll; overflow-y: hidden;"
+                      class="parallel_histograms"
+                      data-histograms="${JSON.stringify(
+                        assessmentScoreHistogramByDate.map(function (day) {
+                          return {
+                            label: day.date_formatted,
+                            mean: day.mean_score_perc,
+                            histogram: day.histogram,
+                          };
+                        }),
```

Comment:
Nit, you can simplify this, though note that the brackets are needed to avoid misinterpreting the braces with new commands.
```suggestion
                        assessmentScoreHistogramByDate.map((day) => ({
                          label: day.date_formatted,
                          mean: day.mean_score_perc,
                          histogram: day.histogram,
                        })),
```